src/Model/Feed.php

```diff
@@ -37,12 +37,14 @@ class Feed implements FeedRequiredInterface
 
     private \DateTimeInterface $updated;
 
+    private Author $author;
```

Comment:
[RFC 4287](https://tools.ietf.org/html/rfc4287#section-4.1.1) Feed elements may contain more than 1 author. An array or collection of Authors is needed

Reply:
resolved in commit 14777d90e512c47f3a9cb3a93f29385f244529de